files/build_templates/swss.service.j2

```diff
@@ -21,6 +21,8 @@ ExecStartPre=/usr/bin/docker exec database redis-cli -n 2 FLUSHDB
 ExecStartPre=/etc/init.d/sxdkernel start
 ExecStartPre=/usr/bin/mst start
 ExecStartPre=/sbin/modprobe i2c-dev
+# i2c_ismt driver resides on the same bus as lpc_i2c, we remove it since its not used in Mellanox box to allow lpc_i2c to be inserted
+ExecStartPre=-/sbin/rmmod i2c_ismt
```

Comment:
SN2100 has Intel Atom chip with SMBus controller, which causes i2c-ismt driver to be inserted during device initialization.

Since we include driver in our [config](https://github.com/Azure/sonic-linux-kernel/blob/73e99e9126c2233a36609275a9e9bd0bb0f3d0f5/config#L3171) it is loaded by default.

Is it ok if I'll remove it from config?